### ISSUE TITLE
co19/LanguageFeatures/nnbd/static_errors_A06_t01

### ISSUE THREAD

**Issue by scheglov** [dart-lang/co19#560]
Analyzer does report an error, it is just not where it is annotated in the test.

```
--- Command "dart2analyzer" (took 20ms):
DART_CONFIGURATION=ReleaseX64NNBD out/ReleaseX64NNBD/dart-sdk/bin/dartanalyzer --enable-experiment=non-nullable --ignore-unrecognized-flags --packages=/b/s/w/ir/cache/builder/sdk/.packages --format=machine --no-hints /b/s/w/ir/cache/builder/sdk/tests/co19/src/LanguageFeatures/nnbd/static_errors_A06_t01.dart

static error failures:
Unexpected static error at line 22, column 9, length 1:
- Had error code COMPILE_TIME_ERROR.NOT_INITIALIZED_NON_NULLABLE_INSTANCE_FIELD.

Missing static error at line 23, column 11, length 3:
- Expected unspecified error code.
- Expected unspecified error message.

--- Re-run this test:
python tools/test.py -n analyzer-asserts-strong-linux co19/LanguageFeatures/nnbd/static_errors_A06_t01
```

**Comment by scheglov** [comment]
Also co19/LanguageFeatures/nnbd/static_errors_A06_t03
Also co19/LanguageFeatures/nnbd/static_errors_A06_t05

**Comment by scheglov** [comment]
Also co19/LanguageFeatures/nnbd/static_errors_A06_t07

**Comment by scheglov** [comment]
Also co19/LanguageFeatures/nnbd/static_errors_A06_t09

**Comment by scheglov** [comment]
Also co19/LanguageFeatures/nnbd/static_errors_A06_t11

**Comment by scheglov** [comment]
Also co19/LanguageFeatures/nnbd/static_errors_A06_t13

**Comment by scheglov** [comment]
Also co19/LanguageFeatures/nnbd/static_errors_A06_t15

**Comment by scheglov** [comment]
Also co19/LanguageFeatures/nnbd/static_errors_A06_t17

**Comment by scheglov** [comment]
Also co19/LanguageFeatures/nnbd/static_errors_A06_t19

**Comment by scheglov** [comment]
Also co19/LanguageFeatures/nnbd/static_errors_A06_t21

**Comment by scheglov** [comment]
Also co19/LanguageFeatures/nnbd/static_errors_A06_t23

**Comment by scheglov** [comment]
Also co19/LanguageFeatures/nnbd/static_errors_A06_t25

**Comment by scheglov** [comment]
Also co19/LanguageFeatures/nnbd/static_errors_A06_t27

**Comment by sgrekhov** [comment]
All mentioned tests fixed by af2f1951f4eb4cf34b2e6a18066225d9b522e5e3
